mage_ai/settings/server.py

```diff
@@ -73,7 +73,11 @@ def get_bool_value(value: str) -> bool:
 HOSTNAME = os.getenv('HOSTNAME')
 REDIS_URL = os.getenv('REDIS_URL')
 SERVER_VERBOSITY = os.getenv('SERVER_VERBOSITY', 'info') or 'info'
-SERVER_LOGGING_FORMAT = os.getenv('SERVER_LOGGING_FORMAT')
+SERVER_LOGGING_FORMAT = os.getenv('SERVER_LOGGING_FORMAT', 'plaintext')
+SERVER_LOGGING_TEMPLATE = os.getenv(
+    'SERVER_LOGGING_TEMPLATE',
+    '[%(levelname)s]:[%(name)s]:%(message)s'
```

Comment:
Please add a comma at the end of the line. Thank you.

Reply:
Done